docs/source/data/data_catalog_yaml_examples.md

```diff
@@ -398,6 +398,58 @@ In this example, Kedro understands that `my_dataframe` is the same dataset in it
 In the pipeline, Kedro uses the `spark.SparkDataset` implementation for saving and `pandas.ParquetDataset`
 for loading, so the first node outputs a `pyspark.sql.DataFrame`, while the second node receives a `pandas.Dataframe`.
```

Comment:
Unable to comment on the specific line above as it's unchanged, but in the catalog example only one entry has a `file_format` value - is this intentional?

Reply:
I think `pandas.ParquetDataset` assumes only `parquet` format: https://github.com/kedro-org/kedro-plugins/blob/0847aecb999683dada6909712e787f6de1fc2414/kedro-datasets/kedro_datasets/pandas/parquet_dataset.py#L80